File: Questões/Linguagem C/1010.c
#include <stdio.h>

int main()
{
    int part1, part2, amount1, amount2;
    float price1, price2;

    scanf("%d %d %f", &part1, &amount1, &price1);
    scanf("%d %d %f", &part2, &amount2, &price2);

    printf("VALOR A PAGAR: R$ %.2f\n", amount1 * price1 + amount2 * price2);

    return 0;
}

File: codes/1000/Python/1010.py
product1 = input().split()
product2 = input().split()

code1, amount1, price1 = product1
code2, amount2, price2 = product2

totalPrice = float(price1) * int(amount1) + float(price2) * int(amount2)

print(f'VALOR A PAGAR: R$ {totalPrice:.2f}')
